### PR TITLE
added space function to art and aprint

### PR DESCRIPTION
**Pull request by codewithnick**
#### Reference Issues/PRs
#241 
#### What does this implement/fix? Explain your changes.

added space parameter in art and aprint functions

### CODE REVIEW

File: art/art.py
Comment (sadrasabouri) [inline art/art.py:128]: Change it to `def aprint(artname, number=1, space=1)` (PEP8)

File: art/art.py
Comment (sadrasabouri) [inline art/art.py:145]: Change it to `artname=artname, number=number, __detailed_return=True, space=space)` and run `autopep8.sh` or `autopep8.bat` for fixing this line.

File: art/art.py
Comment (sadrasabouri) [inline art/art.py:151]: change it to `def art(artname, number=1, __detailed_return=False, space=1):` (PEP8)

File: art/art.py
Comment (sadrasabouri) [inline art/art.py:185]: Change it to `    result = (art_value + " " * space) * number` (PEP8)

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
Thanks for your efforts. There are some points you should take into consideration before checking my comments:

- There is no rush. most of my comments were mentioned by @sepandhaghighi in your last PR but were not taken into action.
- You should update `CHANGELOG.md`, too.
- You should write tests for your new features.
- You can commit on your branch and this PR will be updated. So no need for closing PR after each review and opening new one.

**Comment by sadrasabouri** [comment]
No need for new PR @codewithnick 
Just commit and push your code to your `dev` branch and this PR will be updated.
For tests take a look at [here](https://github.com/sepandhaghighi/art/blob/master/art/test.py#L74C1-L74C4).

File: CHANGELOG.md
Comment (sadrasabouri) [inline CHANGELOG.md:13]: I prefer:
```
- `space` parameter added to `art` function
- `space` parameter added to `aprint` function
```

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
Great. Thanks for changes. I left a minor comment.

**Comment by sadrasabouri** [comment]
I will add tests myself after your last commit. @codewithnick

File: CHANGELOG.md
Comment (sepandhaghighi) [reply CHANGELOG.md:13]: @codewithnick Please address this comment 💯

File: art/art.py
Comment (sepandhaghighi) [inline art/art.py:136]: `:param space: space between arts`

File: art/art.py
Comment (sepandhaghighi) [inline art/art.py:161]: `:param space: space between arts`

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@codewithnick Well done 🔥 
I comment on some minor issues.
Please address them, after that we will merge this PR 🚀

**Review by sadrasabouri — APPROVED** [review]
LGTM. Since I do not have push access to @codewithnick 's fork, I'll open a PR fixing tests afterwards.

**Review by sepandhaghighi — APPROVED** [review]
🚀

**Pull request merged**
